packages/text/src/TextMetrics.ts

```diff
@@ -57,8 +57,8 @@ export class TextMetrics
     public static HEIGHT_MULTIPLIER: number;
 
     // TODO: These should be protected but they're initialized outside of the class.
```

Comment:
Should the new getters be protected, per this comment? They are no longer initialized outside the class.

Reply:
I'll do the cleanup in a new PR. I didn't want to change the API in this one and be 100% backwards, but I agree that it should be fixed. I'll at least update the TODO comment.